pkg/release-controller/release_info.go

```diff
@@ -43,12 +42,12 @@ func NewCachingReleaseInfo(info ReleaseInfo, size int64, architecture string) Re
 				var iArr []BugDetails
 				iArr, err = info.Bugs(parts[1], parts[2])
```

Comment:
Please fix the variable names to reflect the reality of what's actually being returned and used in this method

Reply:
Fixed